docs/maintain/maintain-guides-how-to-nominate-polkadot.md

```diff
@@ -40,143 +41,42 @@ performs due diligence on the validators that they elect. When looking for valid
-to stake and nominate.
-
-The "Payouts" subsection ([link](https://polkadot.js.org/apps/#/staking/payouts)) allows you to
-claim rewards from staking.
-
-The "Targets" subsection ([link](https://polkadot.js.org/apps/#/staking/targets)) will help you
-estimate your earnings and this is where it's good to start picking favorites. For additional
-information on content provided in this table checkout
-[What to take into consideration when nominating](learn-nominator#what-to-take-into-consideration-when-nominating).
-
-The "Waiting" subsection ([link](https://polkadot.js.org/apps/#/staking/waiting)) lists all pending
-validators that are awaiting more nominations to enter the active validator set. Validators will
-stay in the waiting queue until they have enough DOT backing them (as allocated through the
-[Phragmén election mechanism](../learn/learn-phragmen.md)). It is possible validator can remain in
-the queue for a very long time if they never get enough backing.
-
-The "Validator Stats" subsection ([link](https://polkadot.js.org/apps/#/staking/query)) allows you
-to query a validator's stash address and see historical charts on era points, elected stake,
-rewards, and slashes.
+:::info Support Articles
 
-Pick "Account actions", then click the "+ Nominator" button.
+If you need to use the Polkadot-JS UI, clicking on the topics below will take you to the relevant support articles.
 
-You will see a modal window that looks like the below:
-![nominator-update-1](../assets/polkadotjs_nominate_button.png)
-
-Select a "value bonded" that is **less** than the total amount of DOT you have, so you have some
-left over to pay transaction fees. Transaction fees are currently around 0.01 DOT, but they are
-dynamic based on a variety of factors including the load of recent blocks.
-
-Also be mindful of the reaping threshold - the amount that must remain in an account lest it be
-burned. That amount is 1 DOT on Polkadot, so it's recommended to keep at least 1.5 DOT in your
-account to be on the safe side.
-
-Choose whatever payment destination that makes sense to you. If you're unsure, you can choose "Stash
-account (increase amount at stake)" to simply accrue the rewards into the amount you're staking and
-earn compound interest.
-
-![Payout account selection dropdown with the custom account option highlighted](../assets/payout/01.png)
-
-:::note
-
-These concepts have been further explained in Polkadot's
-[UI Walkthrough Video](https://youtu.be/FCXC0CDhyS4?t=219)
+- [How to Bond Tokens and Nominate](https://support.polkadot.network/support/solutions/articles/65000168057-polkadot-js-ui-how-do-i-stake-nominate-on-polkadot-)
+- [How to Select Validators](https://support.polkadot.network/support/solutions/articles/65000150130-how-do-i-know-which-validators-to-choose-)
+- [How to Monitor the Bags List](https://support.polkadot.network/support/solutions/articles/65000181018-i-have-more-than-the-minimum-bonded-but-i-m-not-getting-rewards)
```

Comment:
Probably need to move this article to a different location in the document. 
```suggestion
```

Reply:
Yep, created a section for this